setup.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import sys
+
+python_version = sys.version_info[:2]
+if python_version < (3, 6):
```

Comment:
We support 3.6 in mathics-core right? 

What in 3.7 do we need?

Reply:
Ooops. this refers to the error message below.

Probably it needs to be fixed in mathics-core as well.